openlibrary/templates/account/create.html

```diff
@@ -75,12 +75,14 @@ <h1 class="ol-signup-hero__title">$_("Sign Up")</h1>
             $:field(form.username, suffix=str(screenname_url()))
             $:field(form.password)
 
-            <div class="ol-signup-form__checkbox">
-                $:form.ia_newsletter.render() <label for="ia_newsletter">$:form.ia_newsletter.description</label>
-            </div>
-            <div id="rpd-checkbox" class="ol-signup-form__checkbox">
+           <div class="ol-signup-form__checkbox">
+             <input type="checkbox" id="ia_newsletter" name="ia_newsletter" value="1" />
```

Comment:
```suggestion
             <input type="checkbox" id="ia_newsletter" name="ia_newsletter" />
```
I think that we have code that assumes the [default value](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/checkbox#value_2) will be sent went checkboxes are clicked.